Grillbot/Modules/TempUnverifyModule.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Discord.Commands;
+using Discord.WebSocket;
+using Grillbot.Services;
+using Grillbot.Services.Preconditions;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Grillbot.Modules
+{
+    [IgnorePM]
+    [Group("unverify")]
+    [Name("Odebrání přístupu.")]
+    [RequirePermissions("TempUnverify")]
+    public class TempUnverifyModule : BotModuleBase
+    {
+        private TempUnverifyService Service { get; }
+
```

Comment:
Wouldnt naming it UnverifyService etc. instead of just a Service, make it more readable?